docs/setup/openldap.md

```diff
@@ -31,6 +31,9 @@ sudo ldapadd -Y EXTERNAL -H ldapi:/// -f bootstrap.ldif
 
 If successful, the above command should display: ```adding new entry "olcDatabase=mdb,cn=config"```.
 
+Now, be careful : after a fresh slapd install there's already a olcDatabase={1}mdb.ldif
```

Comment:
I would say instead:

"After a fresh slapd install, you might already have a database set up for you, depending on how the package manager of the distribution is configured, or if the package planned to do so.

To check the database number in your LDAP setup, you can go through the `/etc/ldap/slapd.d/cn=config/` subdirectory and check the files `olcDatabase={x}mdb.ldif`". The different files should contain a `olcSuffix` attribute. The expected geOrchestra database is the one which contains a `olcSuffix: dc=georchestra,dc=org`.

the memberOf.ldif file should match the expected database number in OpenLDAP, and you might need to adapt this LDIF file.
"

Reply:
+1
Indeed, that's THE answer : checking the file that contains right olcSuffix.

I may seem obvious for those who knows ldap, it is not for others, like me...